presto-docker/bin/entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xeu
+
+set -o pipefail
```

Comment:
keep all bash options together:

```suggestion
set -xeuo pipefail
```

and keep only `#!/bin/bash`  on the shebang line

Reply:
👍